frontend/src/components/TreeViewer/TreeViewer.jsx

```diff
@@ -235,18 +240,32 @@ function toggleInfoBoxVisibility(clickedNodeName, curViewingNodeName) {
   return nodeViewingAfterToggle;
 }
 
-export const TreeViewer = ({ data, treeID }) => {
+/**
+ * Gets the arguments of the query string from a given URL
+ * @param {string} name The name of the query-string parameter to get
+ * @param {string} url The url from where to parse query string
+ */
+const getParameterByName = (name, url) => {
+  name = name.replace(/[[\]]/g, "\\$&");
```

Comment:
From @mit1221 : "There already exist functions in JS for doing this: https://www.sitepoint.com/get-url-parameters-with-javascript/. I think it's safer to use this instead." Source: https://github.com/VerWiki/VerWikiApp/pull/22#discussion_r611085214

Reply:
From Sultan: He tried to use the existing functions but they didn't exactly fit our needs; so he had to make a new one